src/harness/fake_sas.py

```diff
@@ -176,7 +176,15 @@ def _GetSuccessResponse(self):
   def _GetMissingParamResponse(self):
     return {'responseCode': MISSING_PARAM}
 
-
+  def InjectEscZone(self, request, ssl_cert=None, ssl_key=None)
+    zoneData = req['zoneDate']
+    if(zoneData['usage'] == 'EXCLUSION_ZONE'):
+      idAttributes = zoneData['id'].split('/')
+      zoneId = ''; 
+     for zoneIdPart in idAttributes[4 : len(idAttributes)]:
+       zoneId+=zoneIdPart
+     return {'zone_id' : zoneId}
```

Comment:
I apologize: my doc is misleading in this regard. I used "zone_id" in the doc but intended to use "zoneId" in the code since that is consistent with the style used in other SAS APIs. Can you please update this accordingly?